raphtory/src/python/filter/property_filter_builders.rs

```diff
@@ -307,40 +410,69 @@ where
     M: Clone + Send + Sync + 'static,
     PropertyFilter<M>: CreateFilter + TryAsCompositeFilter,
 {
-    fn len(&self) -> PyPropertyFilterOps {
+    fn len(&self) -> PyResult<PyPropertyFilterOps> {
```

Comment:
you should make a trait for the aggregation methods on the rust side so you don't have to repeat this implementation for every builder